docs/concepts/nodes.md

```diff
@@ -25,128 +26,66 @@ You're likely to see the term _node_ throughout the IPFS docs, issues, and relat
+- Can be either Kubo or Helia nodes; however there are standalone implementations as well:
+  - [js-libp2p/circuit-relay](https://github.com/libp2p/js-libp2p/blob/master/doc/CONFIGURATION.md#setup-with-relay)
+  - [go-libp2p-relay-daemon](https://github.com/libp2p/go-libp2p-relay-daemon)
+- They're used by both Kubo and Helia nodes.
+    - A Helia _node_ in the browser can't talk TCP, so a relay can help increase the number of _peers_ that can be communicated with. A Helia _node_ with browser-supported [transports](https://github.com/libp2p/js-libp2p/blob/master/doc/CONFIGURATION.md#transport) can talk with a Kubo _peer_ with no overlapping transports using a relay _peer_ if that relay supports one transport from each other _peer_:
+      1. At least one browser-supported transport (appropriate transports are [enabled by default in Helia browser _nodes_](https://github.com/ipfs/helia/blob/d2a928aa1590d5aa642c4c6747d5282f665af43f/packages/helia/src/utils/libp2p-defaults.browser.ts))
+      2. At least one Kubo _peer_ supported transport.
 
 #### Limitations of relay nodes:
-- v1 relays can be used by anyone without any limits, unless [go-libp2p-relay-daemon](https://github.com/libp2p/go-libp2p-relay-daemon) is used with ACLs (Access Control Lists) set up.
 - v2 relays are "limited relays" that are designed to be used for [Direct Connection Upgrade through Relay](https://github.com/libp2p/specs/blob/master/relay/DCUtR.md) (aka hole punching).
 - Not configurable in Kubo; uses a preset list of relays
 
 See [p2p-circuit relay](https://github.com/libp2p/specs/tree/master/relay)
 
 ### Bootstrap
 
-Both Kubo and JS-IPFS nodes use bootstrap nodes to initially enter the DHT.
+Both Kubo and Helia _nodes_ use bootstrap _nodes_ to initially enter the DHT.
 
 #### Features of a bootstrap node:
 
-- All default bootstrap nodes are part of the public DHT.
-- The list of bootstrap nodes a JS-IPFS node connects to is configurable in their config files.
+- All default bootstrap _nodes_ are part of the public DHT.
+- The list of bootstrap _nodes_ a Helia _node_ connects to is controlled by configuring [libp2p's peerDiscovery option](https://github.com/libp2p/js-libp2p/blob/master/doc/CONFIGURATION.md#peer-discovery).
 
 #### Limitations of a bootstrap node:
 
-- If an IPFS node only has one bootstrap node listed in that configuration and that bootstrap node goes offline, the IPFS node will lose access to the public DHT if it were to restart.
+- If an IPFS _node_ only has one bootstrap _node_ listed in that configuration and that bootstrap node goes offline, the IPFS node will lose access to the public DHT if it were to restart.
+  - Note that you should be able to configure your _peer_ store in your implementation to cache healthy connectable _peers_ so you can connect to them again after a restart, instead of bootstrap _nodes_. [Kubo recently added support for this](https://github.com/ipfs/kubo/pull/8856).
```

Comment:
[Kubo has support this](https://github.com/ipfs/kubo/issues/3926)

"recent" is relative
I think better to link to the issue which has the context and links to the PR